### ISSUE TITLE
Cannot install in mac

### ISSUE THREAD

**Issue by mewben** [vova616/chipmunk#8]
```
# github.com/vova616/chipmunk
vova616/chipmunk/collide.go:264: calling method reset with receiver con (type **Contact) requires explicit dereference
```

**Comment by sharpner** [comment]
this library seems to be dead... you can find a workaround here: http://savorywatt.com/2015/02/09/simple-2d-box-physics-in-golang-with-chipmunk/

**Comment by vova616** [comment]
should be be fixed #10

**Comment by sharpner** [comment]
@vova616 are you planning to maintain this repo again? would be awesome!
